.github/workflows/docker.yml

```diff
@@ -0,0 +1,64 @@
+name: Docker Publish
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  # Publish to GitHub Container Registry
+  build-and-push-docker-image:
+    # Master branch only - extra checks
+    if: ${{ github.ref == 'refs/heads/master' }}
+    needs: code-tests
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env: 
+      REGISTRY: ghcr.io
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for OpenActive Test Suite
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REGISTRY }}/openactive/test-suite
+          labels: |
+            org.opencontainers.image.title=OpenActive Test Suite
+            org.opencontainers.image.description=Test suite for OpenActive data publishing and Open Booking API implementations
+            org.opencontainers.image.vendor=OpenActive
+          tags: |
+            latest
+
+      - name: Build and push Docker image for OpenActive Test Suite
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+  # Copied from code-tests.yml, TODO: Remove this duplication
```

Comment:
This is a TODO for later. There are ways to do this but I was thinking of getting it out the door before fixing that